src/kernels/jupyter/connection/jupyterServerProviderRegistry.ts

```diff
@@ -108,7 +108,9 @@ class JupyterUriProviderAdaptor extends Disposables implements IJupyterUriProvid
         const token = new CancellationTokenSource();
         try {
             value = this.provider.extensionId === JVSC_EXTENSION_ID ? value : undefined;
-            const items = await this.provider.commandProvider.getCommands(value || '', token.token);
+            const items = await (
+                this.provider.commandProvider.provideCommands || this.provider.commandProvider.getCommands
```

Comment:
Some extensions like AZ ML and Syunapse are already trying the new Proposed API.
The plan is to remove the old methods in a week, this way others have sufficient time to adopt the new changes